datasets/evidence_infer_treatment/dataset_infos.json

```diff
@@ -0,0 +1 @@
 of different treatments without needing to read the paper.\n", "citation": "@inproceedings{lehman-etal-2019-inferring,\n    title = \"Inferring Which Medical Treatments Work from Reports of Clinical Trials\",\n    author = \"Lehman, Eric  and\n      DeYoung, Jay  and\n      Barzilay, Regina  and\n      Wallace, Byron C.\",\n    booktitle = \"Proceedings of the 2019 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers)\",\n    month = jun,\n    year = \"2019\",\n    address = \"Minneapolis, Minnesota\",\n    publisher = \"Association for Computational Linguistics\",\n    url = \"https://www.aclweb.org/anthology/N19-1371\",\n    pages = \"3705--3717\",\n}\n", "homepage": "https://github.com/jayded/evidence-inference", "license": "", "features": {"Text": {"dtype": "string", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Prompts": {"feature": {"PromptID": {"dtype": "int32", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Outcome": {"dtype": "string", "id": null, "_type": "Value"}, "Intervention": {"dtype": "string", "id": null, "_type": "Value"}, "Comparator": {"dtype": "string", "id": null, "_type": "Value"}, "Annotations": {"feature": {"UserID": {"dtype": "int32", "id": null, "_type": "Value"}, "PromptID": {"dtype": "int32", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Valid Label": {"dtype": "bool", "id": null, "_type": "Value"}, "Valid Reasoning": {"dtype": "bool", "id": null, "_type": "Value"}, "Label": {"dtype": "string", "id": null, "_type": "Value"}, "Annotations": {"dtype": "string", "id": null, "_type": "Value"}, "Label Code": {"dtype": "int32", "id": null, "_type": "Value"}, "In Abstract": {"dtype": "bool", "id": null, "_type": "Value"}, "Evidence Start": {"dtype": "int32", "id": null, "_type": "Value"}, "Evidence End": {"dtype": "int32", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "evidence_inference", "config_name": "1.1", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 55361753, "num_examples": 1931, "dataset_name": "evidence_inference"}, "test": {"name": "test", "num_bytes": 6875650, "num_examples": 240, "dataset_name": "evidence_inference"}, "validation": {"name": "validation", "num_bytes": 7358118, "num_examples": 248, "dataset_name": "evidence_inference"}}, "download_checksums": {"https://github.com/jayded/evidence-inference/archive/v1.1.zip": {"num_bytes": 114452688, "checksum": "945a81cf40665cd797504728858da54dbb39e16a7785bda833f8d475a407a952"}}, "download_size": 114452688, "post_processing_size": null, "dataset_size": 69595521, "size_in_bytes": 184048209}}
```

Comment:
```suggestion
{"2.0": {"description": "Data and code from our \"Inferring Which Medical Treatments Work from Reports of Clinical Trials\", NAACL 2019. This work concerns inferring the results reported in clinical trials from text.\n\nThe dataset consists of biomedical articles describing randomized control trials (RCTs) that compare multiple treatments. Each of these articles will have multiple questions, or 'prompts' associated with them. These prompts will ask about the relationship between an intervention and comparator with respect to an outcome, as reported in the trial. For example, a prompt may ask about the reported effects of aspirin as compared to placebo on the duration of headaches. For the sake of this task, we assume that a particular article will report that the intervention of interest either significantly increased, significantly decreased or had significant effect on the outcome, relative to the comparator.\n\nThe dataset could be used for automatic data extraction of the results of a given RCT. This would enable readers to discover the effectiveness of different treatments without needing to read the paper.\n", "citation": "@inproceedings{lehman-etal-2019-inferring,\n    title = \"Inferring Which Medical Treatments Work from Reports of Clinical Trials\",\n    author = \"Lehman, Eric  and\n      DeYoung, Jay  and\n      Barzilay, Regina  and\n      Wallace, Byron C.\",\n    booktitle = \"Proceedings of the 2019 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers)\",\n    month = jun,\n    year = \"2019\",\n    address = \"Minneapolis, Minnesota\",\n    publisher = \"Association for Computational Linguistics\",\n    url = \"https://www.aclweb.org/anthology/N19-1371\",\n    pages = \"3705--3717\",\n}\n", "homepage": "https://github.com/jayded/evidence-inference", "license": "", "features": {"Text": {"dtype": "string", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Prompts": {"feature": {"PromptID": {"dtype": "int32", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Outcome": {"dtype": "string", "id": null, "_type": "Value"}, "Intervention": {"dtype": "string", "id": null, "_type": "Value"}, "Comparator": {"dtype": "string", "id": null, "_type": "Value"}, "Annotations": {"feature": {"UserID": {"dtype": "int32", "id": null, "_type": "Value"}, "PromptID": {"dtype": "int32", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Valid Label": {"dtype": "bool", "id": null, "_type": "Value"}, "Valid Reasoning": {"dtype": "bool", "id": null, "_type": "Value"}, "Label": {"dtype": "string", "id": null, "_type": "Value"}, "Annotations": {"dtype": "string", "id": null, "_type": "Value"}, "Label Code": {"dtype": "int32", "id": null, "_type": "Value"}, "In Abstract": {"dtype": "bool", "id": null, "_type": "Value"}, "Evidence Start": {"dtype": "int32", "id": null, "_type": "Value"}, "Evidence End": {"dtype": "int32", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "evidence_infer_treatment", "config_name": "2.0", "version": {"version_str": "2.0.0", "description": null, "major": 2, "minor": 0, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 76408584, "num_examples": 2674, "dataset_name": "evidence_infer_treatment"}, "test": {"name": "test", "num_bytes": 9408156, "num_examples": 334, "dataset_name": "evidence_infer_treatment"}, "validation": {"name": "validation", "num_bytes": 10085622, "num_examples": 340, "dataset_name": "evidence_infer_treatment"}}, "download_checksums": {"http://evidence-inference.ebm-nlp.com/v2.0.tar.gz": {"num_bytes": 36528800, "checksum": "6abe0d4ec0d331834981c0171c3c79d47515761867f82f1dc6066e43863a1586"}}, "download_size": 36528800, "post_processing_size": null, "dataset_size": 95902362, "size_in_bytes": 132431162}, "1.1": {"description": "Data and code from our \"Inferring Which Medical Treatments Work from Reports of Clinical Trials\", NAACL 2019. This work concerns inferring the results reported in clinical trials from text.\n\nThe dataset consists of biomedical articles describing randomized control trials (RCTs) that compare multiple treatments. Each of these articles will have multiple questions, or 'prompts' associated with them. These prompts will ask about the relationship between an intervention and comparator with respect to an outcome, as reported in the trial. For example, a prompt may ask about the reported effects of aspirin as compared to placebo on the duration of headaches. For the sake of this task, we assume that a particular article will report that the intervention of interest either significantly increased, significantly decreased or had significant effect on the outcome, relative to the comparator.\n\nThe dataset could be used for automatic data extraction of the results of a given RCT. This would enable readers to discover the effectiveness of different treatments without needing to read the paper.\n", "citation": "@inproceedings{lehman-etal-2019-inferring,\n    title = \"Inferring Which Medical Treatments Work from Reports of Clinical Trials\",\n    author = \"Lehman, Eric  and\n      DeYoung, Jay  and\n      Barzilay, Regina  and\n      Wallace, Byron C.\",\n    booktitle = \"Proceedings of the 2019 Conference of the North {A}merican Chapter of the Association for Computational Linguistics: Human Language Technologies, Volume 1 (Long and Short Papers)\",\n    month = jun,\n    year = \"2019\",\n    address = \"Minneapolis, Minnesota\",\n    publisher = \"Association for Computational Linguistics\",\n    url = \"https://www.aclweb.org/anthology/N19-1371\",\n    pages = \"3705--3717\",\n}\n", "homepage": "https://github.com/jayded/evidence-inference", "license": "", "features": {"Text": {"dtype": "string", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Prompts": {"feature": {"PromptID": {"dtype": "int32", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Outcome": {"dtype": "string", "id": null, "_type": "Value"}, "Intervention": {"dtype": "string", "id": null, "_type": "Value"}, "Comparator": {"dtype": "string", "id": null, "_type": "Value"}, "Annotations": {"feature": {"UserID": {"dtype": "int32", "id": null, "_type": "Value"}, "PromptID": {"dtype": "int32", "id": null, "_type": "Value"}, "PMCID": {"dtype": "int32", "id": null, "_type": "Value"}, "Valid Label": {"dtype": "bool", "id": null, "_type": "Value"}, "Valid Reasoning": {"dtype": "bool", "id": null, "_type": "Value"}, "Label": {"dtype": "string", "id": null, "_type": "Value"}, "Annotations": {"dtype": "string", "id": null, "_type": "Value"}, "Label Code": {"dtype": "int32", "id": null, "_type": "Value"}, "In Abstract": {"dtype": "bool", "id": null, "_type": "Value"}, "Evidence Start": {"dtype": "int32", "id": null, "_type": "Value"}, "Evidence End": {"dtype": "int32", "id": null, "_type": "Value"}}, "length": -1, "id": null, "_type": "Sequence"}}, "length": -1, "id": null, "_type": "Sequence"}}, "post_processed": null, "supervised_keys": null, "builder_name": "evidence_infer_treatment", "config_name": "1.1", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 55361753, "num_examples": 1931, "dataset_name": "evidence_infer_treatment"}, "test": {"name": "test", "num_bytes": 6875650, "num_examples": 240, "dataset_name": "evidence_infer_treatment"}, "validation": {"name": "validation", "num_bytes": 7358118, "num_examples": 248, "dataset_name": "evidence_infer_treatment"}}, "download_checksums": {"https://github.com/jayded/evidence-inference/archive/v1.1.zip": {"num_bytes": 114452688, "checksum": "945a81cf40665cd797504728858da54dbb39e16a7785bda833f8d475a407a952"}}, "download_size": 114452688, "post_processing_size": null, "dataset_size": 69595521, "size_in_bytes": 184048209}}
```
`evidence_inference` -> `evidence_infer_treatment`